### PR TITLE
update to sles15rc2

### PR DESCRIPTION
**Pull request by lucidd**

### CODE REVIEW

File: modules/openstack/base/variables.tf [hunk 1]
```diff
@@ -61,7 +61,7 @@ variable "image_locations" {
   default = {
     centos7 = "http://w3.nue.suse.com/~smoioli/sumaform-images/openstack/centos7.qcow2"
     opensuse423 = "https://download.opensuse.org/repositories/systemsmanagement:/sumaform:/images:/openstack/images/opensuse423.x86_64.qcow2"
-    sles15beta4 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images:/OpenStack/images/sles15beta4.x86_64.qcow2"
+    sles15rc2 = "http://download.suse.de/ibs/Devel:/Galaxy:/Terraform:/Images:/OpenStack/images/sles15rc2.x86_64.qcow2"
```
Comment (moio) [inline modules/openstack/base/variables.tf]: This returns 404, is it expected?

Comment (MalloZup) [reply modules/openstack/base/variables.tf]: yop i think the github code is ok, but i would assume that the image need to be accepted with a SR.

feel free to post here the submission link so we can have a look on it

Comment (mcalmer) [reply modules/openstack/base/variables.tf]: Ahh, you have an extra project for `OpenStack`. AFAIK @lucidd only changed the normal image.

Comment (lucidd) [reply modules/openstack/base/variables.tf]: should be fixed now

**Comment by moio** [comment]
Thanks!

**Pull request merged**